LibraryAPI.java

```diff
@@ -0,0 +1,187 @@
+import java.time.LocalDate;
+import java.util.Arrays;
+
+public class LibraryAPI {
+    public static Author[] getAuthorsData() {
+        Author authors[] = new Author[10];
+        for (int i = 0; i < authors.length; i++) {
+            Author author = new Author(LibraryDataRepo.names[(int) (Math.random() * 7)], LibraryDataRepo.surnames[(int) (Math.random() * 7)], (int) (Math.random() * 100));

```

Comment:
replace hard-coded '7' with length of repo items -> e.g. LibraryDataRepo.names[(int) (Math.random() * LibraryDataRepo.names.length)]